Gibe.DittoProcessors/Processors/StringToMd5HashAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Text;
+using System.Security.Cryptography;
+
+namespace Gibe.DittoProcessors.Processors
+{
+	public class StringToMd5HashAttribute : TestableDittoProcessorAttribute
+	{
+		public override object ProcessValue()
+		{
+			if (Value == null) throw new NullReferenceException();
+
+			var stringValue = Value as string;
+
+			// byte array representation of that string
+			byte[] encodedValue = new UTF8Encoding().GetBytes(stringValue);
+
+			// need MD5 to calculate the hash
+			byte[] hashValue = ((HashAlgorithm)CryptoConfig.CreateFromName("MD5")).ComputeHash(encodedValue);
+
+			// string representation (similar to UNIX format)
+			return BitConverter.ToString(hashValue).Replace("-", string.Empty).ToLower();
```

Comment:
The hash gets returned like so: `07-bd-b0-98-0f-3f-74-13-f0-1a-43-67-d6-bc-61-49`, which is then converted to `07bdb0980f3f7413f01a4367d6bc6149`.